spec/requests/event_attendees_spec.rb

```diff
@@ -14,74 +14,73 @@
 # of tools you can use to make these specs even more expressive, but we're
 # sticking to rails and rspec-rails APIs to keep things simple and stable.
 
-RSpec.describe '/event_attendees', type: :request do
+RSpec.describe "/event_attendees", type: :request do
   let(:event_attendee) { create :event_attendee }
 
   let(:user) { nil }
+
   before do
     sign_in user if user
   end
 
-  describe 'GET /index' do
+  describe "GET /index" do
     subject(:get_index) { get event_attendees_url }
 
-    let!(:event_attendee) { create :event_attendee }
```

Comment:
This let renders the table on the page.

Reply:
But the tests still pass... Is that a smell?  I deleted several of the `let!`s?

Reply:
I was able to resolve the Rubocop error by having the `expect` check the `response.body` for the `profile_id`.  This fixed the cop and still passed.  Does that make sense?

Applied the same logic in `events_spec` and `profiles_spec`.